packages/block-library/src/post-comment-content/block.json

```diff
@@ -8,6 +8,14 @@
 	"textdomain": "default",
 	"usesContext": [ "commentId" ],
 	"supports": {
-		"html": false
+		"color": {
+			"link": true
+		},
+		"typography": {
+			"fontSize": true,
+			"lineHeight": true
+		},
+		"html": false,
+		"__experimentalLayout": true
```

Comment:
I think we will not need to add the `__experimentalLayout` for this block. Layout is for container blocks that enables controlling `width/height`.

Reply:
Thanks, @ntsekouras. I did that because @SantosGuillamot suggested adding that specific setting. I'll talk with him to clarify that.

Reply:
I suggested adding that because it's something supported in the [Post Content block](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/post-content/block.json), and I thought it could be a good idea to follow a similar pattern. It seemed to me that it would add more flexibility for editing the layout, but I don't have a strong opinion here, so totally opened to change it 🙂

Reply:
@scruffian @MaggieCabrera @jffng curious what has come up in themes as uses here.

Reply:
I've not seen a real life use case where we'd want to use this outside of a post comment block.

Reply:
I think `layout` would make more sense in the parent blocks like `Comment Loop`